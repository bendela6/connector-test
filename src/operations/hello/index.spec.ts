import { Route } from "OpenApiRouter";
import { handleHello } from ".";

describe("hello operation", () => {
  it("should return a 200 response", async () => {
    // GIVEN
    const route: Route = {
      route: "/hello",
      operation: {
        operationId: "hello",
        responses: {},
      },
      headers: {},
      pathParameters: {},
    };

    // WHEN
    const response = handleHello(route);

    // THEN
    expect(response).toMatchSnapshot();
  });
});
