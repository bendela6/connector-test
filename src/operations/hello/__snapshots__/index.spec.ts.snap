// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello operation should return a 200 response 1`] = `
Object {
  "body": Object {
    "message": "Hello, world!",
  },
  "status": 200,
}
`;
